lib/inets/test/httpd_bench_SUITE.erl

```diff
@@ -105,37 +100,37 @@ end_per_suite(Config) ->
 
 %% Init keepalive servers group
 init_per_group(Group, Config) when Group == http_dummy_keep_alive;
-				   Group == https_dummy_keep_alive;
-				   Group == http_inets_keep_alive;
-				   Group == https_inets_keep_alive;
-				   Group == http_nginx_keep_alive;
-				   Group == https_nginx_keep_alive ->
+                                   Group == https_dummy_keep_alive;
+                                   Group == http_inets_keep_alive;
+                                   Group == https_inets_keep_alive;
+                                   Group == http_nginx_keep_alive;
+                                   Group == https_nginx_keep_alive ->
     Version = http_version(Group),
     start_web_server(Group,
-		     [{keep_alive, true},
-		      {reuse_sessions, false},
-		      {http_version, Version},
-		      {http_opts,[{version, Version}]},
-		      {http_headers, [{"connection", "keep-alive"}]},
-		      {httpc_opts, [{keep_alive_timeout, 1500},
-				    {max_keep_alive_length, ?config(iter, Config)}]}
-		      | Config]);
+                     [{keep_alive, true},
+                      {reuse_sessions, false},
+                      {http_version, Version},
+                      {http_opts,[{version, Version}]},
+                      {http_headers, [{"connection", "keep-alive"}]},
+                      {httpc_opts, [{keep_alive_timeout, 1500},
+                                    {max_keep_alive_length, ?config(iter, Config)}]}
+                      | Config]);
 %% Init non-keepalive servers group
 init_per_group(Group, Config)  when Group == http_dummy;
-				    Group == https_dummy;
-				    Group == http_inets;
-				    Group == https_inets;
-				    Group == http_nginx;
-				    Group == https_nginx ->
+                                    Group == https_dummy;
+                                    Group == http_inets;
+                                    Group == https_inets;
+                                    Group == http_nginx;
+                                    Group == https_nginx ->
     Version = http_version(Group),
     start_web_server(Group,
-		     [{keep_alive, false},
-		      {reuse_sessions, false},
-		      {http_version, Version},
-		      {http_headers, [{"connection", "close"}]},
-		      {http_opts,[{version, Version}]},
-		      {httpc_opts, [{keep_alive_timeout, 0}, {max_keep_alive_length, 0}]}
-		      | Config]);
+                     [{keep_alive, false},
+                      {reuse_sessions, false},
+                      {http_version, Version},
+                      {http_headers, [{"connection", "close"}]},
+                      {http_opts,[{version, Version}]},
+                      {httpc_opts, [{keep_alive_timeout, 0}, {max_keep_alive_length, 0}]}
+                      | Config]);
```

Comment:
This line is wrongly indented. Erlang mode for emacs indents it differently, that is there is one space too much (and was so from the start).